docs/developer/architecture/logging.rst

```diff
@@ -0,0 +1,235 @@
+Logging
+=======
+
+This section summarizes centralized logging design architecture which is closely
+linked to the above requirements (to be merged).
+
+The following terms will be used throughout the context of this document so it
+is fair that we describe their meanings.
+
+Collector/Shipper
+-----------------
+A log Collector allows you to collect logs from different sources, unify
+and send them to multiple destinations. An example of such a system is the
+Linux utility Rsyslog
+
+Distributor
+-----------
+A log Distributor is a utility capable of adding metadata to logs and
+forwarding these logs to unique sources.
+The metadata (labels) set on log streams are used by log visualizers for
+querying purposes. An example of such system is Logstash.
+
+Visualizer
+----------
+A log Visualizer allows one to query and visualize logs. This can be possible
```

Comment:
Query as a part of the "Visualizer"? That would be part of the Database IMO, or, if we really want to decorrelate them, an extra component (one that would maintain extra indices, for instance).